src/client/containers/OrderConfirmationPage/OrderConfirmationPage.Container.js

```diff
@@ -2,5 +2,30 @@ import React from 'react';
 import './OrderConfirmationPage.styles.css';
 
 export default function OrderConfirmationPage() {
-  return <div>Order Confirmation Page</div>;
+  return <div className="confirmation-page">
+    <div className="inner-div">
+      <div className="wrapper">
+        <div className="wrapper-content">
+          <h3>
+            THANK YOU FOR YOUR ORDER
```

Comment:
Consider using CSS for this instead, such as [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform).